### ISSUE TITLE
Add CUDA 12.6.2

### ISSUE THREAD

**Issue by jakirkham** [rapidsai/build-planning#89]
Recently CUDA 12.6.2 was released. Filing this track supporting it in RAPIDS

**Comment by sisodia1701** [comment]
Waiting for a compiler bug fix.

**Comment by jakirkham** [comment]
We now have CUDA 12.6.1, which includes a fix needed for RAPIDS

Also discussed a pynvjitlink issue we saw in CUDA 12.6 with Graham and Brandon. We concluded that it wasn't a blocker. So pynvjitlink could be updated to work with CUDA 12.6

So this is unblocked. Will defer to others on timing and relative prioritization for including in RAPIDS

Updated the issue based on this information

**Comment by vyasr** [comment]
Closing as superseded by #139.
